[2025-04-15T08:08:36.398+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Anas.consume_and_insert_task manual__2025-04-15T08:08:26.737532+00:00 [queued]>
[2025-04-15T08:08:36.405+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Anas.consume_and_insert_task manual__2025-04-15T08:08:26.737532+00:00 [queued]>
[2025-04-15T08:08:36.405+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-15T08:08:36.405+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-04-15T08:08:36.405+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-15T08:08:36.414+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): consume_and_insert_task> on 2025-04-15 08:08:26.737532+00:00
[2025-04-15T08:08:36.422+0000] {standard_task_runner.py:55} INFO - Started process 465 to run task
[2025-04-15T08:08:36.425+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Anas', 'consume_and_insert_task', 'manual__2025-04-15T08:08:26.737532+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/day.py', '--cfg-path', '/tmp/tmphvw3_osw']
[2025-04-15T08:08:36.427+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask consume_and_insert_task
[2025-04-15T08:08:36.470+0000] {task_command.py:388} INFO - Running <TaskInstance: Anas.consume_and_insert_task manual__2025-04-15T08:08:26.737532+00:00 [running]> on host ac4c8066c1b3
[2025-04-15T08:08:36.520+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Anas
AIRFLOW_CTX_TASK_ID=consume_and_insert_task
AIRFLOW_CTX_EXECUTION_DATE=2025-04-15T08:08:26.737532+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-15T08:08:26.737532+00:00
[2025-04-15T08:08:36.522+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.7', 9092)]>: connecting to kafka:9092 [('172.18.0.7', 9092) IPv4]
[2025-04-15T08:08:36.524+0000] {conn.py:615} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.7', 9092)]>: Broker version identified as 2.6
[2025-04-15T08:08:36.524+0000] {conn.py:456} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.7', 9092)]>: Connection complete.
[2025-04-15T08:08:36.525+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/day.py", line 188, in consume_and_insert
    value_deserializer=lambda x: json.loads(x.decode('utf-8'))
  File "/home/airflow/.local/lib/python3.7/site-packages/kafka/consumer/group.py", line 419, in __init__
    self._subscription.subscribe(topics=topics)
  File "/home/airflow/.local/lib/python3.7/site-packages/kafka/consumer/subscription_state.py", line 149, in subscribe
    self.change_subscription(topics)
  File "/home/airflow/.local/lib/python3.7/site-packages/kafka/consumer/subscription_state.py", line 180, in change_subscription
    ensure_valid_topic_name(t)
  File "/home/airflow/.local/lib/python3.7/site-packages/kafka/util.py", line 57, in ensure_valid_topic_name
    raise TypeError('All topics must not be None')
TypeError: All topics must not be None
[2025-04-15T08:08:36.532+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=Anas, task_id=consume_and_insert_task, execution_date=20250415T080826, start_date=20250415T080836, end_date=20250415T080836
[2025-04-15T08:08:36.539+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 16 for task consume_and_insert_task (All topics must not be None; 465)
[2025-04-15T08:08:36.556+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-04-15T08:08:36.569+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
